packages/mint-components/src/components/sqm-share-code/readme.md

```diff
@@ -15,6 +15,19 @@
 | `tooltiptext`     | `tooltip-text`     | This is shown after someone has successfully copied the cpde to the clipboard.                                          | `string`                                                                                                    | `"Copied to Clipboard"`    |
 
```

Comment:
Flagging that the title of the readme is not correct

Reply:
Changed to `# sqm-share-code`